jupyterhub/apihandlers/users.py

```diff
@@ -18,6 +18,43 @@ def get(self):
         users = self.db.query(orm.User)
         data = [ self.user_model(u) for u in users ]
         self.write(json.dumps(data))
+    
+    @admin_only
+    @gen.coroutine
+    def post(self):
+        data = self.get_json_body()
+        print(data)
+        if not data or not isinstance(data, dict) or not data.get('usernames'):
+            raise web.HTTPError(400, "Must specify at least one user to create")
+        
+        usernames = data.pop('usernames')
+        self._check_user_model(data)
+        
+        for name in usernames:
+            user = self.find_user(name)
+            if user is not None:
```

Comment:
well, the use case I had in mind was someone maintaining a plaintext list of their users (say a class). It's a lot easier to copy/paste the whole file each time you want to make sure everyone has an account and you may have added users, than to manually track your new people in a separate list. For example if you made the list from an alphabetically sorted spreadsheet, having to manually prune it of existing names would be more annoying...